### PR TITLE
edits to mutation appendix

### PR DESCRIPTION
**Pull request by petrelharp**
This addresses comments in #165, hopefully - what do you think, @gregorgorjanc?

### CODE REVIEW

**Comment by petrelharp** [comment]
I also realized we hadn't said anything about the `root_distribution` parameter, so I added a paragraph about that.

File: paper.tex [hunk 1]
```diff
@@ -1737,27 +1737,25 @@ \section*{Mutation generation}
...
-and would require additional bookkeeping necessary to place mutations uniformly on a tree.
+this algorithm is $O(T)$,
+where $T$ is the number of trees.
+An alternative algorithm would apply mutations separately to each tree.
+Since the number of branches per tree is proportional to the number of leaves,
+the simplest implementation would be $O(T S)$,
+where $S$ is the number of samples.
+Any more efficient tree-by-tree algorithm would have to make use of the fact
+that structure is shared between adjacent trees---effectively, therefore,
+working with shared edges.
```
Comment (petrelharp) [inline paper.tex:1751]: thanks; I'm a lot more clear about this now (hopefully the text is also =)

File: paper.tex [hunk 2]
```diff
@@ -1737,27 +1737,25 @@ \section*{Mutation generation}
...
 
 Next, alleles are chosen for each mutation.
 If the site was not previously mutated, then a new ancestral allele is chosen for the site,
 according to an input distribution of ancestral state allele probabilities.
 Then, each mutation on the tree is considered in turn,
-% I don't understand what is meant by "older mutations are considered first"?
-% Aren't past/older mutations irrelevant given the current state (=parental allele) and a transition matrix?
```
Comment (petrelharp) [inline paper.tex:1755]: I just meant that the *parent* has to be chosen (obviously) before the child mutation. In practice, we need to make sure this happens somehow, but we haven't really said how here, so the comment was redundant; I deleted it.

File: paper.tex [hunk 3]
```diff
@@ -1737,27 +1737,25 @@ \section*{Mutation generation}
...
-in order so that older mutations are considered first,
-and a derived allele is chosen based on the parental allele
-(which may be the ancestral allele or the derived allele of a previous mutation)
-% What is meant by "some randomness"?
-and some randomness.
```
Comment (petrelharp) [inline paper.tex:1760]: meaning we choose randomly; again, this is obvious, so I deleted it.

File: paper.tex [hunk 4]
```diff
@@ -1769,8 +1767,20 @@ \section*{Mutation generation}
...
+i.e., the \texttt{root\_distribution}.
+Mutation models with a finite possible set of alleles have a natural choice
+for this distribution---the \emph{stationary distribution} of the mutation process.
+(All mutation models are Markovian,
+so this may be found as the top left eigenvector of the mutation matrix.)
+This is the default in most models,
+except, e.g., the \texttt{BinaryMutationModel}, whose alleles are 0 and 1
+and always labels the ancestral allele ``0''.
+However, mutational processes are not in general stationary,
+so we often allow different root distribution to be specified.
```
Comment (petrelharp) [inline paper.tex:1781]: New paragraph! How is it?

File: paper.tex [hunk 5]
```diff
@@ -1800,9 +1810,9 @@ \section*{Mutation generation}
 However, we have chosen to leave them in, so that for instance
 simulating with the \texttt{HKY} mutation model will result in silent mutations
 if not all equilibrium frequencies are the same.
-Although the presence of silent mutations may at first be surprising
```
Comment (petrelharp) [inline paper.tex:1803]: we had double "although"s here

**Comment by petrelharp** [comment]
See query in #165 about the remaining comment, though.

**Comment by petrelharp** [comment]
Not sure about this last commit - see what you think? Happy to go either way.

File: paper.tex (hunk 1 above)
Comment (jeromekelleher) [reply paper.tex:1751]: I'll probably change the notation a bit here for consistency with the rest of the document, t trees and n samples.

**Review by jeromekelleher — COMMENTED** [review]
Looks good, I'll do a light edit pass now as there's a few small issues.

File: paper.tex (hunk 1 above)
Comment (jeromekelleher) [reply paper.tex:1751]: Ah, using t a second ago for times. I change to \tau. I find capitals weird here - these denote either sets of random variables in my mind.

Comment (jeromekelleher) [reply paper.tex:1751]: Actually, come to think of it again, there's no need for this para and it's just confusing matters, breaking the flow of explanation.

File: paper.tex (hunk 4 above)
Comment (gregorgorjanc) [reply paper.tex:1781]: Great @petrelharp. I know this was already merged - I am just catching up

**Pull request merged**